examples/pytorch/stgcn_wave/load_data.py

```diff
@@ -0,0 +1,45 @@
+import torch
+import numpy as np
+import pandas as pd
+
+
+def load_matrix(file_path):
+    return pd.read_csv(file_path, header=None).values.astype(float)
+
+
+def load_data(file_path, len_train, len_val):
+    df = pd.read_csv(file_path, header=None).values.astype(float)
+    train = df[: len_train]
+    val = df[len_train: len_train + len_val]
+    test = df[len_train + len_val:]
+    return train, val, test
+
+
+def data_transform(data, n_his, n_pred, day_slot, device):
```

Comment:
Maybe add an explanation (either as docstring or comment) what transformation it does?

Reply:
@jianmohuo could you provide more comments in docstring. For example, what are the inputs? what is the type of data? what its dimensions mean?

Reply:
also giving an example of `csv` file would be good.